tools/serverpod_cli/lib/src/migrations/generator.dart

```diff
@@ -101,14 +105,28 @@ class MigrationGenerator {
     );
   }
 
+  Future<List<MigrationVersion>> getAllMigrationVersions(
+    List<String> modules,
+  ) async {
+    var versions = <Future<MigrationVersion?>>[];
+    for (var module in modules) {
+      var moduleVersions = getLatestMigrationVersion(module);
+      versions.add(moduleVersions);
+    }
+    var resolved = await Future.value(versions);
+    return resolved.whereType<MigrationVersion>().toList();
+  }
+
   Future<DatabaseDefinition> _getSourceDatabaseDefinition(
     String? latestVersion,
     int priority,
+    List<DatabaseMigrationVersion> installedModules,
   ) async {
     if (latestVersion == null) {
       return DatabaseDefinition(
         tables: [],
         priority: priority,
+        installedModules: installedModules,
```

Comment:
Suggestion: Make this an empty list.

If we don't find a source database definition we should assume there is no previous migration. And in that case nothing should have been "installed". This should therefore be empty.